src/library/file.c

```diff
@@ -462,11 +462,7 @@ char *get_hash_from_fd2(int fd, size_t size, const int is_sha)
 		if (is_sha) {
 			SHA256(mapped, size, (unsigned char *)&hptr);
 		} else {
-#ifdef USE_DEB
```

Comment:
This should likely use an MD5 preprocessor definition instead of becoming unconditional.